elijah/cards-template1a.js

```diff
@@ -0,0 +1,89 @@
+// Simple version (no error-detection)
+
```

Comment:
Everything is working, looking good. Take a look at your rank and suit functions though and see what you might be able to do to reduce them to something simpler. Are the loops necessary? Nice work here.